src/data/faq.json

```diff
@@ -466,9 +466,9 @@
-                            "On the one hand, the Robert Koch Institute, which is legally and content-wise responsible for its distribution, is responsible for the delivery of the app. At the moment, the focus is only on delivery via the respective App Stores. Furthermore, the app only works with Google Play Services. With a distribution via the Play Store it can be assumed that Google Play Services are installed. We cannot make these assumptions for other distribution channels at the moment.",
-                            "The distribution of the existing app on other channels is currently published as a request on <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/57' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#57</a>. The discussion about alternative distribution platforms such as F-Droid can be found at <a href='https://github.com/corona-warn-app/cwa-documentation/issues/5' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-documentation#5</a>.",
-                            "Furthermore, there is an initiative in the developer community to make the Corona-Warn-App executable independently of Google Play Services via appropriate extensions. Details can be found in the issue <a href='https://github.com/corona-warn-app/cwa-app-android/issues/75' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#75</a>."
+                            "On the one hand, the Robert Koch Institute, which is legally and content-wise responsible for its distribution, is responsible for the delivery of the app. At the moment, the focus is only on delivery via the respective App Stores.",
+                            "The distribution of the existing app on other channels is currently published as a request on <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/57' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#57</a>. The discussion about alternative distribution platforms such as F-Droid can be found at <a href='https://github.com/corona-warn-app/cwa-app-android/issues/1483' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#1483</a>.",
+                            "The developer community has developed an inofficial fork of the app under the name <i>Corona Contact Tracing Germany</i> to make the Corona-Warn-App executable independently of Google Play Services via appropriate extensions. However, the fork is not maintained by the Robert Koch-Institut and therefore receives <b>no official support</b>. Details about the development can be found in the issue <a href='https://github.com/corona-warn-app/cwa-app-android/issues/75' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#75</a> as well as in the code repository of the fork: <a href='https://codeberg.org/corona-contact-tracing-germany/cwa-android' target='_blank' rel='noopener noreferrer'>Corona Contact Tracing Germany</a>."

```

Comment:
Change
"The developer community has developed an inofficial fork of the app under the name Corona Contact Tracing Germany to make the Corona-Warn-App executable independently of Google Play Services via appropriate extensions."

to
"The developer community has developed an inofficial fork of the app under the name Corona Contact Tracing Germany to make the Corona-Warn-App executable independent of Google Play services via appropriate extensions."
- changes "independently" to "independent", you could also say "to make the Corona-Warn-App executable available independently of Google Play Services"
- Services => services (see https://play.google.com/store/apps/details?id=com.google.android.gms&hl=en&gl=US)

Robert Koch-Institut => Robert Koch Institute (in English - see https://www.rki.de/EN/Service/Imprint/imprint_node.html

Reply:
ok, thanks. adopted: "The developer community has developed an inofficial fork of the app under the name Corona Contact Tracing Germany to make the Corona-Warn-App executable independent of Google Play services via appropriate extensions."